src/components/Slider/__tests__/slider-ranged-tests.jest.js

```diff
@@ -0,0 +1,260 @@
+import React from "react";
+import { act, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { snapshotDiff } from "../../../../jest/utils";
+import { renderSliderInRangeMode } from "./slider-tests.utils";
+
+jest.useFakeTimers();
+
+jest.mock("../../TextField/TextField", () => {
```

Comment:
If I understand correctly you mock the implementation so it won't appear in the diff output. 
This means that each time that we'll want to add a new snapshot test, we'll need to replace the internal components like you did here. Is there a way to reduce the code duplication for these cases? Perhaps a helper function that gets a `path` parameter and calls `jest.mock` with a similar implementation like you have here?

Reply:
I mock other components - because I don't want to update Snapshots if there are changes at TextField / Icon component. It not at Slider component responsibility.
And yes, we can think about reuse of components mocks according to your suggestion.